### PR TITLE
improved this garbage

### PR DESCRIPTION
**Pull request by wvaeu**
- reduced recipe word count 
- updated language to follow more common recipe standards 
- added more intuitive sections

### CODE REVIEW

File: seitan.md [hunk 1]
```diff
@@ -1,37 +1,40 @@
...
-   However, I do follow this conventional wisdom.
-5. Stir frying the seitan by itself in some neutral oil is really nice. Deep
-   frying is also quite good. Would not recommend eating the seitan without
-   doing these things because the Maillard reaction definitely improves the
-   flavor noticably.
-6. Put in stir fry, curry, or anything else that you desire.
-
-## Misc
-* When simmering, make sure that the simmer is not too vigorous or else the
-    seitan will not hold its shape as well.
```
Comment (wvaeu) [inline seitan.md:34]: > `  #(Some cooking methods are simmering, steaming, and banking. So far, I have only tried simmmering)`

File: seitan.md [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
+2. Knead all ingredients in a mixing bowl for 5+ minutes. (The more you knead the dough, the chewier the seitan
+   will be)
+3. Separate into 2-3 equal chunks.
+4. Add soy sauce to pot and reduce heat to simmer chunks for 45 minutes. (When simmering, make sure that the simmer is not too vigorous or else the
+   seitan will not hold its shape as well)
+5. Remove seitan from pot, store in fridge. (This should keep in the fridge for a long time. It would last about a week, or more) 
+
+## Cooking methods
+
+* Some cooking methods are simmering, steaming, and banking. So far, I have only tried simmmering.
```
Comment (wvaeu) [inline seitan.md:26]: delete (moved to step 1)

**Review by wvaeu — COMMENTED** [review]
morning edits

**Review by jeffzwang — APPROVED** [review]
LGTM!

**Comment by wvaeu** [comment]
💸🤑💰💵

**Comment by jferguson4** [comment]
Jeff, I found your gps project and saw that aside from school project your intention was same as mine. My cell phone did not give reliable tracks for hikes, so I built one using Ada's ultimate mt3339, the one with the SMD base, I milled the pcb, added an encoder to select function, an E-INK display and li-poly battery, and feather proto m0 to run the code.  case is 3d printed about size of cigarette pack.  works ok, but I think i'll do another one with UBLOX NEO M8N which seems a better GPS.

You must have graduated Harvard by now. Are you in Grad School?    john ferguson   jferg977 (usual) a o l . c o m

**Pull request merged**